Books/book.rb

```diff
@@ -0,0 +1,19 @@
+require_relative '../item'
+
+class Book < Item
+  attr_accessor :publisher, :cover_state, :label_name
+  attr_reader :id, :publish_date, :archived
+
+  # rubocop:disable Metrics/ParameterLists
```

Comment:
- [ ] OPTIONAL
You have **disabled Rubocop checks** on parts of your code. Please enable all Rubocop checks on your code and try modifying that portion of your code to adhere to Rubocop standards. This will ensure a robust and clean code.